shared/react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -69,6 +71,13 @@ protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
 
+        ReactContext currentContext = getReactInstanceManager().getCurrentReactContext();
+        if (currentContext != null) {
+            currentContext
+                    .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                    .emit("androidIntentNotification", getIntent().getExtras().containsKey("notification"));
```

Comment:
we pass a boolean parameter here, but we don't seem to use it when consuming `androidIntentNotification`. Dead code?